ledger-tool/src/program.rs

```diff
@@ -255,9 +257,6 @@ impl fmt::Display for Output {
     }
 }
 
-impl QuietDisplay for Output {}
-impl VerboseDisplay for Output {}
-
 // Replace with std::lazy::Lazy when stabilized.
 // https://github.com/rust-lang/rust/issues/74465
```

Comment:
Looks like have been stabilized, just in case

Reply:
This is unrelated to my change.
I do not understand this code well enough to update it without a non-trivial effort.